db/migrations/20221114104700_create-water-regions.js

```diff
@@ -6,7 +6,7 @@ exports.up = async function (knex) {
   await knex
```

Comment:
I think chaining multiple things using `.then()` should work if we wanted to be consistent about using `return knex` everywhere:
```js
  return knex
    .schema
    .withSchema('water')
    .createTable(tableName, table => {
      ...
    })
    .then(() => {
      knex.raw(`
        ...
      `)
    })
```
But is it actually worth it? 🤷

Reply:
I'll give it a go!